apps/sertra/pages/create.tsx

```diff
@@ -17,13 +17,45 @@ export const getStaticProps: GetStaticProps = async () => {
   };
 };
 
+type Inputs = {
+  serverName: string;
+  connectionInfo: string;
+  description: string;
+  mode: string;
+  mods: [string];
+  isPasswordProtected: string;
+};
+
 const SubmitServer: NextPage<{ swrFallback: { [key: string]: unknown } }> = ({
   swrFallback,
 }) => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Inputs>();
+  const onSubmit: SubmitHandler<Inputs> = (data) => {
+    const prepped_data = {
+      name: data.serverName,
+      description: data.description,
+      community: "v-rising",
+      connection_data: data.connectionInfo,
+      // TODO: Populate this with mod ids etc
+      mods: ["1", "2", "3"],
+      is_pvp: data.mode === "pvp" ? true : false,
+      requires_password: data.isPasswordProtected ? true : false,
```

Comment:
If `data` now implements `Input` interface, then `data.isPasswordProtected` is already a boolean and this ternary operator is redundant.

Reply:
Comment about mods list of string was answer for both 👍 Also good that you pointed out that password field thing, didn't realize that some how 👌